tripper/backends/ontopy.py

```diff
@@ -135,29 +148,32 @@ def parse(self, source=None, location=None, data=None, format=None,
         elif location:
             self.onto.load(filename=location, format=format, **kwargs)
         elif data:
-            #s = io.StringIO(data)
-            #self.onto.load(filename=s, format=format, **kwargs)
+            # s = io.StringIO(data)
+            # self.onto.load(filename=s, format=format, **kwargs)
 
             # Could have been done much nicer if it hasn't been for Windows
             filename = None
             try:
-                kw = {"delete": False}
+                tmpfile_options = {"delete": False}
                 if isinstance(data, str):
-                    kw.update(mode="w+t", encoding=encoding)
-                with tempfile.NamedTemporaryFile(**kw) as f:
-                    f.write(data)
-                    filename = f.name
+                    tmpfile_options.update(mode="w+t", encoding=encoding)
+                with tempfile.NamedTemporaryFile(**tmpfile_options) as handle:
+                    handle.write(data)
+                    filename = handle.name
                 self.onto.load(filename=filename, format=format, **kwargs)
             finally:
                 if filename:
                     os.remove(filename)
 
         else:
-            raise ValueError(
-                "either `source`, `location` or `data` must be given"
-            )
-
-    def serialize(self, destination=None, format='turtle', **kwargs):
+            raise ValueError("either `source`, `location` or `data` must be given")
```

Comment:
Please keep line length below 79 characters

Reply:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
Hehe, reference against reference. For me short lines counts for readability (especially in an 80 character terminal).